Package@swift-5.6.swift

```diff
@@ -0,0 +1,102 @@
+// swift-tools-version:5.6
```

Comment:
To make reviewing this file easier, here's the diff between this file and `Package@swift-5.5.swift`:

```
1c1
< // swift-tools-version:5.5
---
> // swift-tools-version:5.6
19c19
<     .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .exact("0.50500.0")),
---
>     .package(url: "https://github.com/apple/swift-syntax.git", exact: "0.50600.1"),
51c51,52
<         "SwiftSyntax",
---
>         .product(name: "SwiftSyntax", package: "swift-syntax"),
>         .product(name: "SwiftSyntaxParser", package: "swift-syntax"),
69c70,73
<       dependencies: ["SwiftSyntax"],
---
>       dependencies: [
>         .product(name: "SwiftSyntax", package: "swift-syntax"),
>         .product(name: "SwiftSyntaxParser", package: "swift-syntax"),
>       ],
83c87,89
<       dependencies: ["SwiftSyntax"],
---
>       dependencies: [
>         .product(name: "SwiftSyntax", package: "swift-syntax"),
>       ],
```